--- gnuradio-3.7.13.4/gr-blocks/lib/socket_pdu_impl.cc.orig	2018-07-15 18:09:20.000000000 +0200
+++ gnuradio-3.7.13.4/gr-blocks/lib/socket_pdu_impl.cc	2019-04-22 20:56:25.097648774 +0200
@@ -165,7 +165,12 @@
     void
     socket_pdu_impl::start_tcp_accept()
     {
-      tcp_connection::sptr new_connection = tcp_connection::make(d_acceptor_tcp->get_io_service(), d_rxbuf.size(), d_tcp_no_delay);
+#if BOOST_VERSION >= 107000
+#define GET_IO_SERVICE(s) ((boost::asio::io_context&)(s)->get_executor().context())
+#else
+#define GET_IO_SERVICE(s) ((s)->get_io_service())
+#endif
+      tcp_connection::sptr new_connection = tcp_connection::make(GET_IO_SERVICE(d_acceptor_tcp), d_rxbuf.size(), d_tcp_no_delay);
 
       d_acceptor_tcp->async_accept(new_connection->socket(),
         boost::bind(&socket_pdu_impl::handle_tcp_accept, this,
