--- gnuradio/gr-qtgui/lib/Makefile.am~	2011-10-23 20:06:37.000000000 +0200
+++ gnuradio/gr-qtgui/lib/Makefile.am	2012-04-01 16:28:11.377579704 +0200
@@ -92,4 +92,5 @@
 	$(BOOST_THREAD_LIB) 		\
 	$(BOOST_DATE_TIME_LIB) 		\
 	-lstdc++			\
+	-lpython			\
 	$(QT_LIBS)
--- gnuradio/gnuradio-core/src/lib/filter/qa_gri_mmse_fir_interpolator_cc.cc~	2011-10-23 20:06:37.000000000 +0200
+++ gnuradio/gnuradio-core/src/lib/filter/qa_gri_mmse_fir_interpolator_cc.cc	2012-07-11 22:22:47.776970614 +0200
@@ -29,6 +29,7 @@
 #include <qa_gri_mmse_fir_interpolator_cc.h>
 #include <gri_mmse_fir_interpolator_cc.h>
 #include <stdio.h>
+#include <stdint.h>
 #include <cmath>
 #include <stdexcept>
 
