--- gnuradio-3.7.9.2/CMakeLists.txt~	2016-04-11 02:35:01.000000000 +0000
+++ gnuradio-3.7.9.2/CMakeLists.txt	2016-05-24 18:58:53.784633481 +0000
@@ -150,7 +150,7 @@
 set(GR_PKG_DATA_DIR    ${GR_DATA_DIR}/${CMAKE_PROJECT_NAME} CACHE PATH "Path to install package data")
 set(GR_DOC_DIR         ${GR_DATA_DIR}/doc CACHE PATH "Path to install documentation")
 set(GR_PKG_DOC_DIR     ${GR_DOC_DIR}/${CMAKE_PROJECT_NAME}-${DOCVER} CACHE PATH "Path to install package docs")
-set(GR_LIBEXEC_DIR     libexec CACHE PATH "Path to install libexec files")
+set(GR_LIBEXEC_DIR     lib${LIB_SUFFIX} CACHE PATH "Path to install libexec files")
 set(GR_PKG_LIBEXEC_DIR ${GR_LIBEXEC_DIR}/${CMAKE_PROJECT_NAME} CACHE PATH "Path to install package libexec files")
 set(GRC_BLOCKS_DIR     ${GR_PKG_DATA_DIR}/grc/blocks CACHE PATH "Path to install GRC blocks")
 set(GR_THEMES_DIR      ${GR_PKG_DATA_DIR}/themes CACHE PATH "Path to install QTGUI themes")
