diff -ur gnuradio-3.7.9.2.orig/gnuradio-runtime/lib/CMakeLists.txt gnuradio-3.7.9.2/gnuradio-runtime/lib/CMakeLists.txt
--- gnuradio-3.7.9.2.orig/gnuradio-runtime/lib/CMakeLists.txt	2015-08-28 14:47:45.000000000 +0000
+++ gnuradio-3.7.9.2/gnuradio-runtime/lib/CMakeLists.txt	2016-05-23 05:30:48.922358208 +0000
@@ -154,7 +154,7 @@
 
 #need to link with librt on ubuntu 11.10 for shm_*
 if((LINUX) OR (CMAKE_SYSTEM_NAME STREQUAL "kFreeBSD"))
-    target_link_libraries(gnuradio-runtime PUBLIC rt)
+    target_link_libraries(gnuradio-runtime PUBLIC rt pthread)
 endif()
 
 ########################################################################
diff -ur gnuradio-3.7.9.2.orig/gr-audio/lib/CMakeLists.txt gnuradio-3.7.9.2/gr-audio/lib/CMakeLists.txt
--- gnuradio-3.7.9.2.orig/gr-audio/lib/CMakeLists.txt	2015-04-13 14:32:43.000000000 +0000
+++ gnuradio-3.7.9.2/gr-audio/lib/CMakeLists.txt	2016-05-23 05:30:48.922358208 +0000
@@ -96,7 +96,7 @@
 
 if(JACK_FOUND)
     target_compile_definitions(gnuradio-audio PRIVATE -DJACK_FOUND)
-    target_link_libraries(gnuradio-audio PUBLIC JACK::JACK)
+    target_link_libraries(gnuradio-audio PUBLIC JACK::JACK pthread)
     target_sources(gnuradio-audio PRIVATE
         ${CMAKE_CURRENT_SOURCE_DIR}/jack/jack_impl.cc
         ${CMAKE_CURRENT_SOURCE_DIR}/jack/jack_source.cc
diff -ur gnuradio-3.7.9.2.orig/volk/lib/CMakeLists.txt gnuradio-3.7.9.2/volk/lib/CMakeLists.txt
--- gnuradio-3.7.9.2.orig/volk/lib/CMakeLists.txt	2016-03-31 05:23:12.000000000 +0000
+++ gnuradio-3.7.9.2/volk/lib/CMakeLists.txt	2016-05-23 05:30:48.925691583 +0000
@@ -534,7 +534,7 @@
 
 #Add dynamic library
 add_library(volk SHARED $<TARGET_OBJECTS:volk_obj>)
-target_link_libraries(volk ${volk_libraries})
+target_link_libraries(volk ${volk_libraries} m)
 target_include_directories(volk
   PUBLIC $<BUILD_INTERFACE:${PROJECT_BINARY_DIR}/include
   PUBLIC $<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}/include
