diff -urNp -x '*.orig' gnuradio-3.8.0.0.org/gnuradio-runtime/lib/CMakeLists.txt gnuradio-3.8.0.0/gnuradio-runtime/lib/CMakeLists.txt
--- gnuradio-3.8.0.0.org/gnuradio-runtime/lib/CMakeLists.txt	2019-08-09 23:40:34.000000000 +0200
+++ gnuradio-3.8.0.0/gnuradio-runtime/lib/CMakeLists.txt	2021-03-05 21:17:40.083297715 +0100
@@ -236,7 +236,7 @@ endif(HAVE_WINDOWS_H)
 
 #need to link with librt on ubuntu 11.10 for shm_*
 if((LINUX) OR (CMAKE_SYSTEM_NAME STREQUAL "kFreeBSD"))
-    target_link_libraries(gnuradio-runtime PUBLIC rt)
+    target_link_libraries(gnuradio-runtime PUBLIC rt pthread)
 endif()
 
 ########################################################################
diff -urNp -x '*.orig' gnuradio-3.8.0.0.org/gr-audio/lib/CMakeLists.txt gnuradio-3.8.0.0/gr-audio/lib/CMakeLists.txt
--- gnuradio-3.8.0.0.org/gr-audio/lib/CMakeLists.txt	2019-08-09 23:40:34.000000000 +0200
+++ gnuradio-3.8.0.0/gr-audio/lib/CMakeLists.txt	2021-03-05 21:17:40.083297715 +0100
@@ -79,7 +79,7 @@ find_package(JACK)
 
 if(JACK_FOUND)
     target_compile_definitions(gnuradio-audio PRIVATE -DJACK_FOUND)
-    target_link_libraries(gnuradio-audio PUBLIC JACK::JACK)
+    target_link_libraries(gnuradio-audio PUBLIC JACK::JACK pthread)
     target_sources(gnuradio-audio PRIVATE
         ${CMAKE_CURRENT_SOURCE_DIR}/jack/jack_impl.cc
         ${CMAKE_CURRENT_SOURCE_DIR}/jack/jack_source.cc
diff -urNp -x '*.orig' gnuradio-3.8.0.0.org/volk/lib/CMakeLists.txt gnuradio-3.8.0.0/volk/lib/CMakeLists.txt
--- gnuradio-3.8.0.0.org/volk/lib/CMakeLists.txt	2019-08-09 23:40:35.000000000 +0200
+++ gnuradio-3.8.0.0/volk/lib/CMakeLists.txt	2021-03-05 21:17:40.083297715 +0100
@@ -566,7 +566,7 @@ target_include_directories(volk_obj
 
 #Add dynamic library
 add_library(volk SHARED $<TARGET_OBJECTS:volk_obj>)
-target_link_libraries(volk ${volk_libraries})
+target_link_libraries(volk ${volk_libraries} m)
 target_include_directories(volk
   PUBLIC $<BUILD_INTERFACE:${PROJECT_BINARY_DIR}/include>
   PUBLIC $<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}/include>
